lib/matplotlib/figure.py

```diff
@@ -2219,6 +2219,12 @@ def __init__(self, parent, subplotspec, *,
         self._set_artist_props(self.patch)
         self.patch.set_antialiased(False)
 
+    def get_size_inches(self):
```

Comment:
This needs to be added to the `figure.pyi` file.

How odd do we think that it is for `SubFigure` to have `get_size_inches` but not `set_size_inches`?  I'm leaning towards that it is OK, with a small thought of "we should implement `set_size_inches` that raises and says you have to go talk to the (grand-)parent figure.

Reply:
I actually think that this should be private (and have a similar private method for the top level Figure that the public `get_size_inches` calls, and then uses where we need the subfigure size can call the private method instead.